yarn-project/bootstrap.sh

```diff
@@ -104,7 +104,7 @@ function test_cmds {
   for test in {prover-node,p2p,ethereum,aztec}/src/**/*.test.ts; do
     if [[ "$test" =~ testbench ]]; then
       # Testbench runs require more memory and CPU.
-      echo "$hash ISOLATE=1 CPUS=18 MEM=12g yarn-project/scripts/run_test.sh $test"
+      echo "$hash ISOLATE=1 CPUS=20 MEM=16g yarn-project/scripts/run_test.sh $test"
```

Comment:
20 cores and 16GB is only 800MB per core. How memory hungry is this test? It might make sense to give it more memory per core.

Reply:
It's 20 p2p clients, will see how this goes and bump accordingly